api/handler/v1/v1.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	"context"
+	"github.com/odpf/salt/server"
+
+	shieldv1 "github.com/odpf/shield/api/protos/github.com/odpf/proton/shield/v1"
```

Comment:
@krtkvrm We can use proton gen code from buf build remotely and avoid keeping protos and gen code locally. cc @rohilsurana

Reply:
Looks like [the proto](https://github.com/odpf/proton/pull/45) has not been merged to master at `proton` yet. Unless that is done we won't have this proto in the buf registry.

Have received feedback that we can't use it unless there is support for branches. Will check with buf folks on that.